styles.css

```diff
@@ -490,3 +410,78 @@ body {
 .description-item {
   color: #344563;
 }
+
+.contact-container {
+  background: url(Assets/contactmebg.png) no-repeat center, #6070ff;
+  background-position: 180px 20px;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  border-top-left-radius: 100px;
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  flex-direction: column;
+  height: 850px;
+  align-items: center;
+  color: #fff;
+  text-align: center;
+}
+
+.c-title-container {
+  font-size: 17px;
+  line-height: 20px;
+  padding-top: 100px;
+}
+
+.c-desc-container {
+  width: 80%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 15px;
+  width: 100%;
+  text-align: center;
+  margin: 0 220px 50px 20px;
+}
+
+.customer-info {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  color: #172b4d;
+  padding: 10px;
+  width: 80%;
+}
+
+.customer-message {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-size: 15px;
+  color: #b3bac5;
+  padding: 10px;
+  width: 80%;
+}
+
+.submit {
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  cursor: pointer;
+  text-decoration: none;
+  padding: 10px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+}
+
+.submit:hover {
+  scale: 1.2;
+}
```

Comment:
**OPTIONAL**- Please implement the interactions for the buttons such as hover, as seen in the Figma template below 🙏. Remember replicating exact Figma specs gives confidence to recruiters in your ability to replicate exact design specs. 👍 . Please try to fix this.

Figma template -
![figma 1button](https://user-images.githubusercontent.com/73607512/192776111-5777e314-d1f0-452a-824c-a449f3467da4.PNG)

Reply:
@mckent05 Thanks for the suggestion. Actually, I didn't notice the above section in the template and was wondering how to find the button interaction. Many thanks for the guidance.